src/index.js

```diff
@@ -43,6 +44,9 @@ export default function algoliaAlexaAdapter (opts) {
   skill.handler = function(event, context) {
     const alexa = Alexa.handler(event, context);
     alexa.appId = alexaAppId;
+    if (languageStrings) {
+      alexa.resources = languageStrings;
```

Comment:
according to the Alexa docs ( I didn't read them fully ) the `resources` is an object, where `languageStrings` is a key, right?